kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/rest/jaxb/DynamicJaxbContext.java

```diff
@@ -338,7 +343,7 @@ private void removeClassFromJaxbContextClassList( List<Class> classList, Illegal
                String className = classLocs.listIterator(classLocs.size()).previous().toString();
                Class removeClass = null;
                try {
-                   removeClass = Class.forName(className);
+                   removeClass = projectClassLoader.loadClass(className);
```

Comment:
out of curiosity as I don't know it myself - what is better to be used:
classLoader.loadClass(className) or Class.forName(className, init, classLoader)?